data_pipelines_cli/airbyte_utils.py

```diff
@@ -48,42 +55,68 @@ def create_update_connections(self) -> None:
             [task.update(self.env_replacer(task)) for task in self.airbyte_config["tasks"]]
             self.update_file(self.airbyte_config)
 
+    def get_workspace_id(self) -> str:
+        workspaces = self.request_handler("workspaces/list").get("workspaces")
+        if not workspaces:
+            raise AirbyteNoWorkspaceConfiguredError(
+                f"No workspaces found in {self.airbyte_url} instance."
+            )
+
+        return workspaces[0].get("workspaceId")
```

Comment:
Why the first workspace is hardcoded? I think the workspace should be taken from the configuration.